tests/integ/modin/window/test_rolling.py

```diff
@@ -126,6 +126,30 @@ def test_rolling_series(window, min_periods, center, agg_func):
             )
 
 
+@agg_func
+@min_periods
+def test_rolling_time_period(agg_func, min_periods):
+    native_df = native_pd.DataFrame(
+        {"B": [0, 1, 2, np.nan, 4]},
+        index=[
+            native_pd.Timestamp("20130101 09:00:00"),
+            native_pd.Timestamp("20130101 09:00:02"),
+            native_pd.Timestamp("20130101 09:00:03"),
+            native_pd.Timestamp("20130101 09:00:04"),
+            native_pd.Timestamp("20130101 09:00:06"),
+        ],
+    )
+    snow_df = pd.DataFrame(native_df)
+    eval_snowpark_pandas_result(
+        snow_df,
+        native_df,
+        lambda df: getattr(
+            df.rolling(window="2s", min_periods=min_periods),
```

Comment:
can we parametrize by some more windows, like `"-2s"`, much smaller windows, and much larger windows?